test/k23si/3SITxnTest.cpp

```diff
@@ -1368,15 +1451,17 @@ seastar::future<> testScenario02() {
                     auto mtr = dto::K23SI_MTR {
                             .timestamp = std::move(ts),
                             .priority = dto::TxnPriority::Medium};
+                    K2LOG_D(log::k23si, "using mtr={}", mtr);
 
-                    return doWrite(k4, v1, mtr, k4, collname, false, true, ErrorCaseOpt::NoInjection);
-                })
-                .then([&, mtr] (auto&& response) {
-                    auto& [status, resp] = response;
-                    K2EXPECT(log::k23si, status, dto::K23SIStatus::Created);
+                    return doWrite(k4, v1, mtr, k4, collname, false, true, ErrorCaseOpt::NoInjection)
+                    .then([&, mtr] (auto&& response) {
+                        auto& [status, resp] = response;
+                        K2EXPECT(log::k23si, status, dto::K23SIStatus::Created);
+                        K2LOG_D(log::k23si, "using mtr={}", mtr);
```

Comment:
there is some whitespace change here. Can you plz fix that to make sure it matches the rest of the code